pravega/zookeeper-operator
# Issue 98: Switch to go 1.13.8 and use gomodules

Signed-off-by: anisha.kj <anisha.kj@dell.com>

# Purpose of the change
Fixes #98
# what the code does:
Updated GO version to `1.13.8`, that will use GO modules. This will allow us to build code outside `GOPATH`

# How to verify it :
1) All the make commands in the make file are working as expected
2) Checkout and build code outside $GOPATH should work.
3)  All basic operator functionality - Cluster Creation, Deletion, Scaling-up & Down, pod Deletion, Upgrades should work as expected
